### PR TITLE
updating air-gap install guide

### PR DESCRIPTION
**Pull request by hubvu**
- Updated the Air Gap install guide.
  - added a prerequisite section.
  - Changed the layout and order of steps to be clearer and consistent with the rest of the install guides.

### CODE REVIEW

File: docs/install/airgapped-clusters.md [hunk 1]
```diff
@@ -1,259 +1,299 @@
 ---
-title: "Airgapped installation"
-weight: 50
+title: "Air-Gap Install"
```
Comment (cannischan) [inline docs/install/airgapped-clusters.md:2]: I would like customer success to agree on what term we should go with

File: docs/install/airgapped-clusters.md [hunk 2]
```diff
@@ -1,259 +1,299 @@
...
 
-- Install the `storageos` kubectl plugin
-- Generate yaml manifests and amend for your use case
-- Pull OCI images to private registries.
-- Install Ondat on your cluster
+> 💡 This guide is recommended for **advanced users** who have experience and permissions to be able to manage air-gapped deployments in their environment. The full procedure for this deployment method is estimated to take ~60 minutes to complete.
 
-There are the following sets of images to pull that will be shown along the
-following steps.
+Below is a quick summary of the procedure that will be covered in this guide;
```
Comment (cannischan) [inline docs/install/airgapped-clusters.md:13]: : instead of ;

Comment (cannischan) [reply docs/install/airgapped-clusters.md:13]: I like this summary

File: docs/install/airgapped-clusters.md [hunk 3]
```diff
@@ -1,259 +1,299 @@
...
 
-```bash
-curl -sSLo kubectl-storageos.tar.gz \
-    https://github.com/storageos/kubectl-storageos/releases/download/v1.1.0/kubectl-storageos_1.1.0_linux_amd64.tar.gz \
-    && tar -xf kubectl-storageos.tar.gz \
-    && chmod +x kubectl-storageos \
-    && sudo mv kubectl-storageos /usr/local/bin/ \
-    && rm kubectl-storageos.tar.gz
-```
+> ⚠️ Make sure you have met the minimum resource requirements for Ondat to successfully run. Review the main [Ondat prerequisites](/docs/prerequisites/) page for more information.
```
Comment (cannischan) [inline docs/install/airgapped-clusters.md:23]: Make sure you have met the minimum resource requirements for Ondat so your set up would be successful. Review the Prerequisites page for more information.

File: docs/install/airgapped-clusters.md [hunk 4]
```diff
@@ -1,259 +1,299 @@
...
-(AWS) or `standard` (GCE). Or you can create the `local-path` StorageClass.
-It's recommended to give Etcd a backend disk that can sustain at least 800
-IOPS. It's best to use provisioned IOPS when possible, otherwise make sure the
-size of the disk is big enough to fulfil the IOPS requirement when performance
-depends on IOPS per GB.  For example, AWS would require a burstable EBS bigger
-than 256G to fulfil the IOPS requirement.
+> ⚠️ Make sure you have a running Kubernetes cluster with a minimum of 3 worker nodes and the sufficient [Role-Based Access Control (RBAC)](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) permissions to deploy and manage applications in the cluster.
 
-1. Create StorageClass (if you don't have one available)
+> ⚠️ Make sure your Kubernetes cluster uses a Linux distribution that is officially supported by Ondat as your node operating system and has the required LinuxIO related kernel modules are available for Ondat to run successfully.
```
Comment (cannischan) [inline docs/install/airgapped-clusters.md:33]: I would suggest to split the following out:
> Make sure your cluster has the required LinuxIO related kernel modules

File: docs/install/airgapped-clusters.md [hunk 5]
```diff
@@ -1,259 +1,299 @@
...
-    ...
+    kubectl get pod --namespace=local-path-storage
+    kubectl get storageclass
     ```
 
-    Define the `storge` size of the Etcd volumes. The backend disk requires at
-    least 800 IOPS for etcd to work normally. If you are on a cloud provider
-    that enables IOPSxGB, it is recommended to use a big enough volume. For
-    instance, on AWS a 256GiB or a 50GiB GCE SSD persistent disk.
+> ⚠️ The `local-path` StorageClass is only recommended for **non production** clusters, as this stores all the data of the `etcd` peers locally, which makes it susceptible to state being lost on node failures.
```
Comment (cannischan) [inline docs/install/airgapped-clusters.md:86]: non-production ?

Comment (cannischan) [reply docs/install/airgapped-clusters.md:86]: susceptible to its state being lost

File: docs/install/airgapped-clusters.md [hunk 6]
```diff
@@ -1,259 +1,299 @@
...
+grep --extended-regexp "RELATED|image:" *.yaml
 ```
 
-## Step 5. Install Ondat
+### Step 5 - Modify Deployment Manifests
 
-```bash
-# Create the operators and CRDs first
-find . -name '*-operator.yaml'  | xargs -I{} kubectl create -f {}
+ 1. **`etcd-operator`** - Modify the `2-etcd-operator.yaml` manifest and apply the following changes.
```
Comment (cannischan) [inline docs/install/airgapped-clusters.md:166]: is this supposed to be 1, 2, 3 (so it should be 1.1.1.?)

**Review by cannischan — COMMENTED** [review]
Made some comments

File: docs/install/airgapped-clusters.md (hunk 2 above)
Comment (hubvu) [reply docs/install/airgapped-clusters.md]: fixed

File: docs/install/airgapped-clusters.md (hunk 6 above)
Comment (hubvu) [reply docs/install/airgapped-clusters.md:168]: fixed this and the indentation for the code snippets, thanks!

File: docs/install/airgapped-clusters.md [hunk 7]
```diff
@@ -1,259 +1,299 @@
...
+ cd storageos-dry-run/
+ ls
+ ```
 
-1. Amend the file `storageos-operator.yaml`
+#### Option B - Using An External `etcd` Deployment
+
+##### Setup An  `etcd`  Cluster
+
+* Ensure that you have an  `etcd`  cluster deployed first before installing Ondat. For instructions on how to set up an external  `etcd`  cluster, review the  [`etcd`  documentation](https://docs.ondat.io/docs/prerequisites/etcd/#production---etcd-on-external-virtual-machines)  page.
```
[the remark this answers is not in the document]
Comment (hubvu) [reply docs/install/airgapped-clusters.md]: fixed

File: docs/install/airgapped-clusters.md (hunk 5 above)
Comment (hubvu) [reply docs/install/airgapped-clusters.md]: fixed thanks!

File: docs/install/airgapped-clusters.md (hunk 4 above)
Comment (hubvu) [reply docs/install/airgapped-clusters.md]: sure, I have made the split to;

> ⚠️ Make sure your Kubernetes cluster uses a Linux distribution that is officially supported by Ondat as your node operating system.
> 
> ⚠️ Make sure that the node operating system have the required LinuxIO related kernel modules are available for Ondat to run successfully.

Let me know if this is fine.

File: docs/install/airgapped-clusters.md (hunk 3 above)
Comment (hubvu) [reply docs/install/airgapped-clusters.md]: fixed.

File: docs/install/airgapped-clusters.md (hunk 4 above)
Comment (cannischan) [reply docs/install/airgapped-clusters.md]: LGTM

**Review by cannischan — APPROVED** [review]
lgtm

**Pull request merged**